tests/test_timesteps.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+
+
+def test_timesteps(test_mp):
+    timesteps = test_mp.timesteps()
+    obs = timesteps[timesteps.category == 'YEARS']
```

Comment:
what is the intention of a category `YEARS`?

Reply:
as discussed renamed to COMMON